deploy/role.yaml

```diff
@@ -191,6 +191,7 @@ rules:
   verbs:
   - get
   - list
+  ${OPERATOR_ROLE_PATCH}
```

Comment:
I used this ${OPERATOR_ROLE_PATCH} to be consistent with the other cases.

Reply:
Ah, yes. Good catch. This is to support view-only for the operator.